### PR TITLE
Add message component from deep-ui

### PR DESCRIPTION
**Pull request by priyesh777**
## Changes:
- Add message component from deep-ui
- Add SVG component from deep-ui
- Add Logo component that will be required in the message component

## This PR doesn't introduce any:

- [x] temporary files, auto-generated files or secret keys
- [x] build works
- [x] eslint issues
- [x] typescript issues
- [x] `console.log` meant for debugging
- [x] typos
- [x] unwanted comments

## This PR contains valid:

- [x] UI changes

### CODE REVIEW

File: src/components/Message/styles.css [hunk 1]
```diff
@@ -0,0 +1,32 @@
+.message {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    padding: var(--dui-spacing-medium);
```
Comment (samshara) [inline src/components/Message/styles.css]: use an appropriate variable. We need to replace`--dui--` variable with the corresponding `--tui--` variable

File: src/components/PendingMessage/index.tsx [hunk 2]
```diff
@@ -20,7 +21,7 @@ function PendingMessage(props: PendingMessageProps) {
         <Cover className={_cs(styles.pendingMessage, className)}>
             <PendingAnimation />
             <div className={styles.message}>
-                { message }
+                {message}
```
Comment (samshara) [inline src/components/PendingMessage/index.tsx:24]:
```suggestion
                { message }
```

File: src/components/QuickActionButton/styles.css [hunk 3]
```diff
@@ -0,0 +1,15 @@
+.button {
+    &.round-button {
+        padding: var(--dui-spacing-extra-small);
```
Comment (samshara) [inline src/components/QuickActionButton/styles.css]: replace CSS  variable with appropriate value from toggle-ui

File: src/components/SelectInput/SearchSelectInput.tsx [hunk 4]
```diff
@@ -25,7 +25,7 @@ function Option(props: OptionProps) {
                 <MdCheck />
             </div>
             <div className={styles.label}>
-                { children }
+                {children}
```
Comment (samshara) [inline src/components/SelectInput/SearchSelectInput.tsx]:
```suggestion
                { children }
```

File: src/components/SelectInput/SearchSelectInput.tsx [hunk 5]
```diff
@@ -41,46 +41,46 @@ export type SearchSelectInputProps<
...
+            | 'onSearchTextChange'
+            | 'dropdownShown'
+            | 'onDropdownShownChange'
+            | 'focused'
+            | 'onFocusedChange'
+            | 'focusedKey'
+            | 'onFocusedKeyChange'
+            | 'hasValue'
+        >
+    ) & (
```
Comment (samshara) [inline src/components/SelectInput/SearchSelectInput.tsx]: keep the format of code the same as previous.

File: src/components/SelectInput/SearchSelectInput.tsx [hunk 6]
```diff
@@ -41,46 +41,46 @@ export type SearchSelectInputProps<
...
+        { nonClearable: true; onChange: (newValue: T, name: K) => void }
+        | { nonClearable?: false; onChange: (newValue: T | undefined, name: K) => void }
+    );
```
Comment (samshara) [inline src/components/SelectInput/SearchSelectInput.tsx]: same as above

File: src/components/SelectInput/SearchSelectInput.tsx [hunk 7]
```diff
@@ -90,8 +90,8 @@ function SearchSelectInput<
     // eslint-disable-next-line @typescript-eslint/ban-types
     O extends object,
     P extends Def,
->(
-    props: SearchSelectInputProps<T, K, O, P, never>,
+    >(
+        props: SearchSelectInputProps<T, K, O, P, never>,
```
Comment (samshara) [inline src/components/SelectInput/SearchSelectInput.tsx]: same as above

File: src/components/Message/index.tsx [hunk 8]
```diff
@@ -0,0 +1,171 @@
...
+import Button from '../Button';
+import QuickActionButton from '../QuickActionButton';
+import PendingMessage from '../PendingMessage';
+
+import styles from './styles.css';
+
+export const genericMemo: (<T>(c: T) => T) = memo;
+
+export interface Props {
+    className?: string;
```
Comment (priyesh777) [inline src/components/Message/index.tsx:15]:
```suggestion
export interface Propz {
    className?: string;
```

File: src/components/SelectInput/SearchSelectInput.tsx (hunk 4 above)
Comment (priyesh777) [reply src/components/SelectInput/SearchSelectInput.tsx]: I consulted this with safar dai and he recommended that removing the spaces would be better.

**Review by samshara — APPROVED** [review]

**Pull request merged**